server/src/domain/system-config/system-config.core.ts

```diff
@@ -44,6 +44,13 @@ export const defaults = Object.freeze<SystemConfig>({
     [QueueName.THUMBNAIL_GENERATION]: { concurrency: 5 },
     [QueueName.VIDEO_CONVERSION]: { concurrency: 1 },
   },
+  machineLearning: {
+    enabled: process.env.IMMICH_MACHINE_LEARNING_URL !== 'false',
```

Comment:
Currently, typesense has the environment variable `TYPESENSE_ENABLED`. Can we use `MACHINE_LEARNING_ENABLED` to be more consistent ?

Reply:
Definitely. It'll just be a breaking change for those who have it set currently.

Reply:
I think this is a change we should make as well and this seems like a good moment to do it.